imperial.js

```diff
@@ -296,15 +298,15 @@ class Imperial {
   }
 
   nations() {
-    let nations = {};
-    ["AH", "IT", "FR", "GB", "GE", "RU"].map((nation) => {
-      nations[nation] = {
+    const nations = new Map()
+    for (const nation of Nation) {
+      nations.set(nation, {
         controller: this.getController(nation, this.log),
         treasury: this.getTreasury(nation, this.log),
         taxChartPosition: this.getTaxChartPosition(nation),
         powerPoints: this.getPowerPoints(nation),
-      };
-    });
+      });
+    };
```

Comment:
This is an interim change. I hope to eventually make a "Record" type that is similar to `Enum` in its strictness, except it stores key-value pairs of string-anything instead of storing a set of strings.